### ISSUE TITLE
Attic Generating an entirely duplicated repository when run as a cron job

### ISSUE THREAD

**Issue by jlowrey** [jborg/attic#314]
So a strange issue I'm having.  Originally I initialized and created a few backups manually from terminal.  After having success doing so, ensuring that the backups were successful and restorable I decided to automate the process.  I created a shell script to store the backups in the same repository and prune using the default in the attic example.  I added a cron job to run the script nightly.  At first I received the attic  error: cache initialization aborted.  I was able to correct this by piping echo "y" into the attic create command in the cron job the first time it was executed.  What happened next is what baffles me.  Attic analyzed all the previous backups and began the backup in the same repository but instead of seeing all the files as duplicates it found none of them as a duplicate and is essentially creating a whole new backup in the repository from scratch.  I am currently waiting for that backup to finish but figured I would go ahead and find out if this the intended behavior?

This is probably not a huge problem for some but our file server currently holds 7TB of data and design projects and it takes approx 3 - 3.5 days for a full backup from scratch to complete. 

I am not an advanced linux user and am obviously new to attic so if there is any additional information I need to provide please let me know and I will do my best to comply.  The server is currently running Ubuntu 14.

Thanks,
Jon

**Comment by ThomasWaldmann** [comment]
@jlowrey did you find out why it happend / how did you solve?

**Comment by jlowrey** [comment]
It was symptoms of something else going on. The backup was running much slower and initially I had to pipe an additional parameter that would allow it to run the first time from a cron job(its somewhere here in the forums but I have forgotten what it was).  When I checked the NAS unit after the initial run via cron it had doubled in space and there was a copy of the repository.  In actuality a default setting on the NAS to automatically archive everything on the NAS was causing the duplication, not attic creating an additional backup. It was a mistake on my part which is why I closed this.  Thanks for the follow-up though.

Thanks,
Jon
